### PR TITLE
Add induction motor models

### LINKED ISSUE #200: [Model/ Test case Name]: Induction motors
### Do you want to add a new model or test case?

- [X] New Model
- [ ] Test Cases

### Briefly describe about the contribution

This issues adds a simple single-cage induction motor model and the WECC fifth order motor model.

### PR DESCRIPTION
**Pull request by FredericSabot**
Closes #200

### CODE REVIEW

**Review by Lampros-Cresym — COMMENTED** [review]
Thank you Frederic for contributing to Colib. 
Do you think it would be possible to add indicative parameters to the tables?

Apart from that both two entries are well described and ready to be approved.

**Review by Lampros-Cresym — APPROVED** [review]

File: pages/models/motors/SimpleMotor/index.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
+---
+layout: page
+title: Single-cage induction motor model
+tags: ["#200", "Induction motor", "RMS", "Dynawo", "Opensource"]
+date: 09/08/2024
+last-updated: 09/08/2024
+authors: Frédéric Sabot (ULB)
+reviewers:
```
Comment (GJCRESYM) [inline pages/models/motors/SimpleMotor/index.md:8]:
```suggestion
reviewers: Lampros Papangelis (CRESYM)
```

File: pages/models/motors/WECC_Motor/index.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
+---
+layout: page
+title: Induction motor model with rotor flux dynamics
+tags: ["#200", "Induction motor", "RMS", "Dynawo", "Opensource"]
+date: 12/08/2024
+last-updated: 12/08/2024
+authors: Frédéric Sabot (ULB)
+reviewers:
```
Comment (GJCRESYM) [inline pages/models/motors/WECC_Motor/index.md:8]:
```suggestion
reviewers: Lampros Papangelis (CRESYM)
```

File: pages/models/motors/WECC_Motor/index.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+---
+
+
+## Context
+
+Motors are a particular kind of load that can account for a large share of the total load especially in industrialised countries. Adequate representation of motors is important, especially in short-term voltage stability studies as motors can cause fault-induced delayed voltage recovery.
```
Comment (GJCRESYM) [inline pages/models/motors/WECC_Motor/index.md:14]:
```suggestion
Motors are a particular kind of load that can account for a large share of the total load especially in industrialised countries. Adequate representation of motors is important, especially in short-term voltage stability studies as motors can cause fault-induced delayed voltage recovery [[1]](#1).
```

File: pages/models/motors/WECC_Motor/index.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+This model has been successfully implemented in :
+
+| Software      | URL | Language | Open-Source License | Last consulted date | Comments |
+| --------------| --- | --------- | ------------------- |------------------- | -------- |
+|Dynawo|[Link](https://github.com/dynawo/dynawo/blob/master/dynawo/sources/Models/Modelica/Dynawo/Electrical/Machines/Motors/MotorFifthOrder.mo)| modelica | [MPL v2.0](https://www.mozilla.org/en-US/MPL/2.0/)  | 12/08/2024 | no comment |
+
+
+## Table of references
+
+<a id="1">[1]</a> PowerWorld. "Load Characteristic MOTORW", https://www.powerworld.com/WebHelp/Content/TransientModels_HTML/Load%20Characteristic%20MOTORW.htm, Last accessed 12/08/2024
```
Comment (GJCRESYM) [inline pages/models/motors/WECC_Motor/index.md:97]: Please remove last accessed date: 

image

**Review by GJCRESYM — CHANGES_REQUESTED** [review]
@FredericSabot : Small changes to be implemented. Could you please have a look?

**Review by Lampros-Cresym — APPROVED** [review]

**Review by GJCRESYM — APPROVED** [review]

**Pull request merged**